tests/test_views.py

```diff
@@ -16,7 +16,10 @@ class ViewsTest(TestCase):
     @classmethod
     def setUpTestData(cls):
         """Create test client, user, posts with different published dates."""
-        cls.user = get_user_model().objects.create()
+        cls.user = get_user_model().objects.create_user(
+            username='test user',
+            password='test password',
+        )
```

Comment:
Is this change actually necessary for the tests to work?

Reply:
I deleted it for now

Reply:
resolved.